### ISSUE TITLE
Document the meaning of "default" and "auto" in extrusion width settings

### ISSUE THREAD

**Issue by alranel** [slic3r/Slic3r#4613]
willmore on IRC asks:

> How come external perimeters can be a value or "auto", but perimeters can only be a value or "default"?

This is not clearly explained in the tooltips and the documentation.

**Comment by kajsilander78** [comment]
Agree, would like to have a formula for auto in tooltip and have a number for default, like default(42).
